### PR TITLE
Make CFP workflow work & store the data in the backend

### PR DESCRIPTION
**Pull request by mzdaniel**
Found the CPF page interaction was not working properly. This PR straighten the data flow in a way we can at least start committing data.

### CODE REVIEW

File: pybay/forms.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ def save_to_models(self):
         except User.DoesNotExist:
 
             # Create a new user
-            password = ''.join(random.choices(string.ascii_uppercase + string.digits, k=10))
+            password = ''.join(random.choice([string.ascii_uppercase + string.digits for i in range(10)]))
```
Comment (sesas) [inline pybay/forms.py:42]: These two lines don't see equivalent.
The first one is choosing 10 letters from 'ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789'.
The second one is choosing 1 string from a 10 item list whose items is the string 'ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789'

```
In [7]: random.choice([string.ascii_uppercase + string.digits for i in range(10)])
Out[7]: 'ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789'
```

Comment (sesas) [reply pybay/forms.py:42]: maybe `sample` is what you are looking for: 
```
In [11]: random.sample(string.ascii_uppercase + string.digits, k=10)
Out[11]: ['K', 'R', '9', 'N', 'M', 'I', 'E', 'Z', '7', '8']
```

Comment (sesas) [reply pybay/forms.py:42]: Also, if we pick the password on behalf of the user, how is that user going to be able to sing-in next time and modify what he submitted?

**Review by sesas — COMMENTED** [review]
I made a couple of comments on this PR @mzdaniel. Let me know what you think of it.

File: pybay/forms.py (hunk 1 above)
Comment (mzdaniel) [reply pybay/forms.py:42]: @sesas:  Good catch. It's just a typo (the list comprehension is meant for the joint method, as it turns out choice does not accept generator expressions.
Aside of this, your random.sample is more elegant and python 3.5 compatible. We'll include it in the next commit.

Comment (sesas) [reply pybay/forms.py:42]: 👍

**Pull request merged**